### PR TITLE
feat: Interviewbit

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Changes proposed

- **Added Interviewbit Object in online-code-editors.json File in frontend Category for Learn and Practice Programming with Different Programming Languages using Online IDE Editor**

## Note to reviewers

- **Additionally, This Interviewbit also Provides Bunch of Topics like DSA, Cheatsheets, Interview Questions**

### CODE REVIEW

**Comment by NarendraKoya999** [comment]
Hi @CBID2 . I have raised PR. Please check and Review on It. If Merged, Please add appropriate Labels to that PR

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -110,5 +110,12 @@
     "url": "https://www.codewars.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "Interviewbit",
+    "description": "Everything you need to crack your Next Tech Interview",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:116]: [Typo]
```suggestion
    "description": "Everything you need to crack your next tech interview is provided in this website",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions on how to improve your descriptions @NarendraKoya999

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**